### ISSUE TITLE
Is IRSA supported for s3 blob stores ?

### ISSUE THREAD

**Issue by cebidhem** [sonatype/helm3-charts#107]
Hi team,

I was wondering if IRSA is a supported auth method for AWS S3 blob stores ? If not, do you have any plan of supporting it ?

Thank you!

**Comment by dervos** [comment]
Yes, this is supported. The blob store integration will look for AWS credentials. In 'values.yaml' you can specify the serviceaccount (that has the required IAM roles attached) or specify the IAM role in the annotations.

**Comment by azhurbilo** [comment]
@dervos it will be great to share working example with specific nexus docker image version 🙏 

cause in our case EKS IRSA is not working too with new version of nexus docker image (aws sdk version).
similar like in this issue https://github.com/Oteemo/charts/issues/281#issuecomment-907870377
